src/pt/spectralscan/src/eu/kanade/tachiyomi/extension/pt/spectralscan/NexusScan.kt

```diff
@@ -0,0 +1,311 @@
+package eu.kanade.tachiyomi.extension.pt.spectralscan
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import rx.Observable
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class NexusScan : HttpSource() {
+
+    // Mantém o ID original para manter a compatibilidade com a extensão anterior (SpectralScan)
```

Comment:
Prefer English code comments, or something that doesn't require translation.
```suggestion
    // SpectralScan (pt-BR) -> Nexus Scan (pt-BR)
```